### PR TITLE
Update LinkedFileDiffMergingTests for top-level statements

### PR DESCRIPTION
**Pull request by sharwell**
See #44423

### CODE REVIEW

File: src/Workspaces/CoreTest/LinkedFileDiffMerging/LinkedFileDiffMergingTests.TextMerging.cs [hunk 1]
```diff
@@ -44,15 +44,15 @@ public void TestIsolatedChangesInBothFiles()
...
         public void TestIdenticalEditAfterIsolatedChanges()
         {
             TestLinkedFileSet(
-                "a b c d e",
-                new List<string> { "a zzz c xx e", "a b c xx e" },
-                @"a zzz c xx e",
+                "a; b; c; d; e;",
+                new List<string> { "a; zzz; c; xx; e;", "a; b; c; xx; e;" },
+                @"a; zzz; c; xx; e;",
                 LanguageNames.CSharp);
```
Comment (CyrusNajmabadi) [inline src/Workspaces/CoreTest/LinkedFileDiffMerging/LinkedFileDiffMergingTests.TextMerging.cs:56]: i'm a little curious about what broke here and why we need the input to the test to change.

Comment (sharwell) [reply src/Workspaces/CoreTest/LinkedFileDiffMerging/LinkedFileDiffMergingTests.TextMerging.cs:56]: It's based on the C# syntax tree, and the previous input didn't parse consistently with the new feature.

**Comment by sharwell** [comment]
@CyrusNajmabadi for review

**Pull request merged**